### ISSUE TITLE
Rewrite all element descriptions to use normative language

### ISSUE THREAD

**Issue by AmeliaBR** [w3c/svgwg#111]
Whenever something describes how an element should appear/behave, need normative language on user agents to make it so.

E.g.,
"Elements that are descendants of a ‘defs’ are not rendered directly"
to
"User agents must not render elements that are descendants of a 'defs', directly."

**Comment by AmeliaBR** [comment]
That was just one example. The issue is more general.

**Comment by dirkschulze** [comment]
@AmeliaBR Independent that the proposed style is preferable, do you think we need to fix that in SVG 2.0?

**Comment by AmeliaBR** [comment]
No, it isn't a priority compared to actual technical changes. The SVG 2.1 (or whatever's next) milestone is appropriate.
